src/ts/components/subtitleoverlay.ts

```diff
@@ -174,6 +285,21 @@ class ActiveSubtitleManager {
   }
 
   /**
+   * Returns the label associated with an already added cue.
+   * @param event
+   * @return {SubtitleLabel}
+   */
+  getCue(event: SubtitleCueEvent): SubtitleLabel {
```

Comment:
Change to `getCues()` and apply style to all cues that match the calculated id, not just the first one.